src/Cache/Engine/ApcEngine.php

```diff
@@ -16,9 +16,10 @@
 
 use APCUIterator;
 use Cake\Cache\CacheEngine;
+use RuntimeException;
 
 /**
- * APC storage engine for cache
+ * APCu storage engine for cache
  */
 class ApcEngine extends CacheEngine
```

Comment:
Should this not be named ApcuEngine, then have a class_alias from ApcEngine?

Reply:
Could do that, as APC is kind of done now.

Reply:
I can do that in a separate PR after this is done.